modules/Bio/EnsEMBL/G2P/DBSQL/GenomicFeatureDiseasePanelAdaptor.pm

```diff
@@ -169,6 +175,12 @@ sub update {
   if (!ref($user) || !$user->isa('Bio::EnsEMBL::G2P::User')) {
     die('Bio::EnsEMBL::G2P::User arg expected');
   }
+  # to check for the visibility 
+  if ($gfd_panel->{confidence_category_attrib} == 88 || $gfd_panel->{confidence_category_attrib} == 89) {
+    $gfd_panel->{is_visible} = 0;
+  } else {
```

Comment:
You have to remove the `else { $gfd_panel->{is_visible} = 1; }` otherwise the `confidence` and `is_visible` are always updated together. 
For example, an entry has confidence 'definitive' but I want to set the visibility to restrictive - this code does not let me do that.